scripts/openzfs-exceptions.txt

```diff
@@ -86,10 +88,13 @@
 6418 131cc95
 6421 ca0bf58
 6434 472e7c6
+6459 - 			changes don't work properly with ZoL codebase
```

Comment:
It appears that most (maybe all) of the resulting warnings are in Linux specific areas.  So we'll want to address them which pulling in the cstyle.pl updates.

All of those `vdev_raidz_math_ssse3.c` warnings should be resolvable with the following small change.  I suspect we'll need to make the same change in other places for the other optimization raidz implementations.

```diff
diff --git a/module/zfs/vdev_raidz_math_ssse3.c b/module/zfs/vdev_raidz_math_sss
index cebb0fe..1742115 100644
--- a/module/zfs/vdev_raidz_math_ssse3.c
+++ b/module/zfs/vdev_raidz_math_ssse3.c
@@ -419,7 +419,8 @@ const raidz_impl_ops_t vdev_raidz_ssse3_impl = {
 #if defined(HAVE_SSSE3) || defined(HAVE_AVX2) || defined(HAVE_AVX512BW)
 
 const uint8_t
-__attribute__((aligned(256))) gf_clmul_mod_lt[4*256][16] = {
+__attribute__((aligned(256))) gf_clmul_mod_lt[4*256][16] =
+{
        { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00  },
        { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

There may be other places where complicated macros are used where you'll need to disable the style checking entirely.  You can do this by wrapping the block with the following comments.

```
/* BEGIN CSTYLED */
...
/* END CSTYLED */
```